src/Modules/OrchardCore.Commerce.Payment.Stripe/Assets/Scripts/stripe-payment-form.js

```diff
@@ -7,8 +7,8 @@ window.stripePaymentForm = function stripePaymentForm(
     errorText,
     missingText,
     updatePaymentIntentUrl,
-    validateUrl = 'checkout/validate/Stripe',
-    paramsUrl = 'checkout/params/Stripe',
+    validateUrl = 'checkout/validate/stripe',
+    paramsUrl = 'checkout/stripe/params',
```

Comment:
All Stripe minimal API endpoint are `~/checkout/stripe/{action}` because they are only Stripe related and other providers probably won't support those. So I thought for the least amount of inconsistency I would do this for all stripe endpoints.
What do you think?

Reply:
But I might revert this for regular Controllers.

Reply:
Why do you want `~/checkout/stripe/{action}`? If you want to make it clear that they are only Stripe related, then you don't need the `checkout` in the path, which indicates the general checkout flow. My point was that if the path starts with `~/checkout/` then it should be structured consistently with the other `~/checkout` actions in `PaymentController`. You can either revert or remove the `checkout/` from the start of the Stripe-only paths, either is fine. Just don't use `~/checkout/stripe` because `stripe` is not an action.